src/ManageCourses.ApiClient/Generated/ManageCoursesApiClient.cs

```diff
@@ -193,18 +193,84 @@ public System.Threading.Tasks.Task<OrganisationCourses> Data_ExportByOrganisatio
     
         /// <summary>Imports the data.</summary>
         /// <exception cref="SwaggerException">A server side error occurred.</exception>
-        public System.Threading.Tasks.Task Data_ImportAsync(Payload payload)
+        public System.Threading.Tasks.Task Data_ImportAsync(UcasPayload payload)
         {
             return Data_ImportAsync(payload, System.Threading.CancellationToken.None);
         }
     
         /// <summary>Imports the data.</summary>
         /// <exception cref="SwaggerException">A server side error occurred.</exception>
         /// <param name="cancellationToken">A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
-        public async System.Threading.Tasks.Task Data_ImportAsync(Payload payload, System.Threading.CancellationToken cancellationToken)
+        public async System.Threading.Tasks.Task Data_ImportAsync(UcasPayload payload, System.Threading.CancellationToken cancellationToken)
```

Comment:
Do we actually need this to be commited?

Reply:
nope thx - fixed